pkg/rancher-desktop/backend/kube/lima.ts

```diff
@@ -339,13 +339,13 @@ export default class LimaKubernetesBackend extends events.EventEmitter implement
   /**
    * Write the openrc script for k3s.
    */
-  protected async writeServiceScript(cfg: BackendSettings, allowSudo: boolean) {
+  protected async writeServiceScript(cfg: BackendSettings, desiredVersion: semver.SemVer, allowSudo: boolean) {
     const config: Record<string, string> = {
       PORT:            this.desiredPort.toString(),
       ENGINE:          cfg.containerEngine.name ?? ContainerEngine.NONE,
       ADDITIONAL_ARGS: `--node-ip ${ await this.vm.ipAddress }`,
       LOG_DIR:         paths.logs,
-      USE_CRI_DOCKERD: BackendHelper.requiresCRIDockerd(cfg.containerEngine.name, cfg.kubernetes.version).toString(),
+      USE_CRI_DOCKERD: BackendHelper.requiresCRIDockerd(cfg.containerEngine.name, cfg.kubernetes.version || desiredVersion.version).toString(),
```

Comment:
```suggestion
      USE_CRI_DOCKERD: BackendHelper.requiresCRIDockerd(cfg.containerEngine.name, desiredVersion.version).toString(),
```
We never care what version is in the settings, just what version will actually get used, right?